ui/src/LandingPage.js

```diff
@@ -113,6 +135,12 @@ class LandingPage extends Component {
     });
   };
 
+  openFulfilmentTriggerDialog = () => {
```

Comment:
a nice small/easy/simple enhancement might be to call `this.getFulfilmentTrigger();` every time that the user opens the dialog box, so that it refreshes the data from the backend